usaspending_api/api_contracts/contracts/v2/autocomplete/accounts/aid.md

```diff
@@ -65,16 +66,16 @@ List of potential Agency Identifiers
 Each component listed here may be omitted, null, or a string value.  If omitted, no filtering will be performed on that component.  If null, the filter will include account numbers missing that component.  If a string, the filter will perform an exact match on account numbers where that component matches the string provided except in the case of aid where a partial match will be performed on the aid component starting with the provided value.
 
 + `ata` (optional, string, nullable)
-    Allocation Transfer Agency Identifier (3 characters). TAS only.
+    Allocation Transfer Agency Identifier (020). TAS only.
```

Comment:
The intent of these descriptions is to define the exact requirements of what will and won't work for this API request, whereas the "Body" sections (see lines 20 and 34) demonstrate examples like this. This change would actually REDUCE the amount of information provided to the user, since it would be unclear if, for example, 883 would be a valid `ata` to provide.

Reply:
Ok

Reply:
Ok

Reply:
I saw a Brian from making it look so difficult it's not that difficult at all Is It Anyway you can call me 7026659915